DOCS/xml-fields.md

```diff
@@ -44,17 +44,19 @@ Within the .xml file, the first and last lines of the file should be the followi
 The main tag used to define the license or exception is, unsurprisingly, **`<license>`** or **`<exception>`**. All of the remaining content will be enclosed within a `<license></license>` or `<exception></exception>` pair of tags.
 
 There are two mandatory attributes for every `<license>` and `<exception>` tag:
-* `licenseId`: the unique SPDX Identifier for the license; should be identical to the filename
-* `name`: the longer "real name" of the license
+
+- `licenseId`: the unique SPDX Identifier for the license; should be identical to the filename
+- `name`: the longer "real name" of the license
 
 There are two additional attributes which are optional, but highly encouraged:
-* `isOsiApproved` (for licenses, not exceptions): either "true" or "false" based on whether this license has been approved by the [Open Source Initiative](https://opensource.org/licenses/alphabetical)
-* `listVersionAdded`: in which release version of the SPDX License List was the license first added, e.g. "3.19".
-  * Typically you'll check the currently-released version at https://spdx.org/licenses/ and increment the minor version by 1 for a new license.
+
+- `isOsiApproved` (for licenses, not exceptions): either "true" or "false" based on whether this license has been approved by the [Open Source Initiative](https://opensource.org/licenses/alphabetical)
+- `listVersionAdded`: in which release version of the SPDX License List was the license first added, e.g. "3.19".
+  - Typically you'll check the currently-released version at https://spdx.org/licenses/ and increment the minor version by 1 for a new license.
 
 Finally, if the license ID has been deprecated, two additional attributes should be included:
```

Comment:
Note that this line should also be changed to reference "one additional attribute" instead of "two additional attributes"